client/state/data-layer/wpcom/read/site/index.js

```diff
@@ -0,0 +1,15 @@
+/**
```

Comment:
> I think we can ditch the external deps block if there aren't any (linter shouldn't complain).

In general I don't think i agree with this comment.  Adding/removing the external/internal blocks just feels like a waste of time considering it will all get removed by uglify anyway.  I figure its best to just leave it there once its been added.